### PR TITLE
Use pip 1.5.6

### PR DESCRIPTION
**Pull request by rhcarvalho**
Second try...

### CODE REVIEW

**Comment by mfojtik** [comment]
[test]

**Comment by mfojtik** [comment]
@bparees @rhcarvalho is looking into this issue. In worst case I think we can live with more latest 'pip' in centos. We already have that situation with nodejs (the RHEL has 0.10.25 and Centos7 has 0.10.35).

**Comment by rhcarvalho** [comment]
Looks like it was just the problem with `pip show`. Now we do not use it anymore.
@mfojtik @bparees PTAL.

**Comment by mfojtik** [comment]
@rhcarvalho test passed, which is good. @bparees I will leave this merge to you :-)

File: 3.3/.sti/bin/run [hunk 1]
```diff
@@ -2,11 +2,11 @@
 source /opt/openshift/etc/generate_container_user
 
 function is_gunicorn_installed() {
-  pip show -q gunicorn
+  hash gunicorn &>/dev/null
```
Comment (bparees) [inline 3.3/.sti/bin/run]: I think "which" is the more traditional test.

Comment (rhcarvalho) [inline 3.3/.sti/bin/run]: http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script/677212#677212

`hash` is a bash builtin and has its advantages over `which`. Agree with you, `which` was the first thing in my mind too.

Comment (bparees) [inline 3.3/.sti/bin/run]: interesting, thanks for the education!

**Pull request merged**